docker/Makefile

```diff
@@ -0,0 +1,53 @@
+git_describe := $(shell git describe)
+vcs_ref := $(shell git rev-parse HEAD)
+build_date := $(shell date -u +%FT%T)
+hadolint_available := $(shell hadolint --help > /dev/null 2>&1; echo $$?)
+hadolint_command := hadolint --ignore DL3008 --ignore DL3018 --ignore DL4000 --ignore DL4001
+hadolint_container := hadolint/hadolint:latest
+
+ifeq ($(IS_NIGHTLY),true)
+	dockerfile := Dockerfile.nightly
+	version := puppet6-nightly
```

Comment:
Since the later `push` only varies by the version, I would set a var here like this:

```make
container_version := latest
```

Reply:
But if the container has `IS_LATEST` set we push both the numbered tag and the latest tag in separate push steps. I'm not following how this would help.

Reply:
Yeah, sorry. I completely misread there and my comment is nonsensical.